src/closure/Job.php

```diff
@@ -8,15 +8,14 @@
 namespace yii\queue\closure;
 
 use SuperClosure\Serializer;
-use yii\base\Object;
 use yii\queue\JobInterface;
 
 /**
  * Closure Job
  *
  * @author Roman Zhuravlev <zhuravljov@gmail.com>
  */
-class Job extends Object implements JobInterface
+class Job implements JobInterface
```

Comment:
This is no equivalent.

Reply:
The job is used in the closure behavior only. It isn't used outside.